lib/challenge.rb

```diff
@@ -16,21 +21,12 @@ def each
       next_fib = current_fib + previous_fib
     end
   end
-
-  attr_reader :stop
-  private :stop
```

Comment:
I didn't see it as a requirement and it didn't seem necessary as long as `@stop` is read-only.